scripts/env/check-package-manager

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+/**
+ * Check that corepack is enabled. This makes sure the correct package manager
+ * is used.
+ *
+ * If not, ask the user to run `corepack enable`.
+ */
+
+// no external deps can be used here, because this runs before deps are installed
+const { execSync } = require('child_process');
+const { stdout } = require('process');
+const { colourise, log, warn } = require('../log');
+
+// we can't use chalk, because this runs before deps are installed
+// https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797
+const reset = '\x1b[0m';
+const blue = '\x1b[34m';
+const red = '\x1b[31m';
+const dim = '\x1b[2m';
+
+// get the package manager and version we need from package.json
+const [packageManager, version] =
+	require('../../package.json').packageManager.split('@');
```

Comment:
This could fail if someone removes the field from `package.json` and I’m not sure they would get a useful error, but that’s quite a small edge case.

Reply:
good point, will add a check